packages/wonder-blocks-layout/src/components/media-layout.tsx

```diff
@@ -209,7 +204,7 @@ class MediaLayoutInternal extends React.Component<CombinedProps, State> {
         return mockStyleSheet;
     }
 
-    render(): React.ReactNode {
+    renderContent(initialRender: boolean): React.ReactNode {
```

Comment:
**note:** This is the old render function but with a parameter to indicate whether it's the initial render or not. This way we can use the same code and just invoke it from the `WithSSRPlaceholder`.